gadopt/energy_solver.py

```diff
@@ -38,12 +32,16 @@
     "ksp_rtol": 1e-5,
     "pc_type": "sor",
 }
-"""Default iterative solver parameters for solution of energy equation. Configured to use the GMRES Krylov scheme
-   with Successive Over Relaxation (SOR) preconditioning. Note that default energy solver parameters
-   can be augmented or adjusted by accessing the solver_parameter dictionary, for example:
-   energy_solver.solver_parameters['ksp_converged_reason'] = None
-   energy_solver.solver_parameters['ksp_rtol'] = 1e-4
-   G-ADOPT defaults to iterative solvers in 3-D.
+"""Default iterative solver parameters for solution of energy equation.
+
+Configured to use the GMRES Krylov scheme with Successive Over Relaxation (SOR)
+preconditioning. Note that default energy solver parameters can be augmented or adjusted
+by accessing the solver_parameter dictionary, for example:
+energy_solver.solver_parameters['ksp_converged_reason'] = None
+energy_solver.solver_parameters['ksp_rtol'] = 1e-4
+
+Note:
+  G-ADOPT defaults to iterative solvers in 3-D.
```

Comment:
Does it also default to iterative solvers in 2-D cylinders? I know this comment it old, but perhaps worth updating to be correct.

Reply:
There is a difference between the transport and Stokes solvers: the former do not consider the model's geometry when choosing their solver strategy. This choice is likely a consequence of the easier energy solves compared to Stokes's. Should this behaviour change?

Reply:
I think it is better if all solvers transition to iterative under the same conditions. Let me think about this and perhaps it can be a future pull request, but leave your comments as is for now then.

Reply:
@drhodrid What is your update?

Reply:
Ohh I didn't realise I needed to give one!! Leave it as it is for now - and we can look at this in a subsequent pull request.